step1_get_eia_demand_data.ipynb

```diff
@@ -91,56 +89,48 @@
    "outputs": [],
    "source": [
     "# Query EIA to get list of regions for which hourly electricity deman data is available\n",
-    "def get_regions_data(ID=2122628):\n",
-    "\n",
-    "    regions_query = urllib.request.urlopen('http://api.eia.gov/category/?api_key={}&category_id={}&format=json'.format(EIA_API_KEY, ID))\n",
+    "def get_regions_data(ID='region-data', data_type='facets[type][]=D&', start='2016-01-01T00', end='2016-01-02T00'):\n",
```

Comment:
Is the time range limit here possibly why you are not finding data for some subregions? The subregions did not exist in 2016. Was it only forecast data for the subregions that was missing?

Reply:
Only forecast. I couldn't even figure out what the right path for the forecast subregion data would be from the dashboard https://www.eia.gov/opendata/browser/electricity/rto/region-sub-ba-data